core-bundle/src/Resources/contao/pages/PageRegular.php

```diff
@@ -808,6 +811,48 @@ protected function createFooterScripts($objLayout)
 			}
 		}
 
+		// Add search index meta data
+		if ($objPage !== null)
+		{
+			$noSearch = (bool) $objPage->noSearch;
+
+			// Backwards compatibility: Do not use $GLOBALS['TL_NOINDEX_KEYS'] anymore
+			// but make sure your page type delivers the correct meta robots tag
+			foreach (array_keys($_GET) as $key)
+			{
+				foreach ($GLOBALS['TL_NOINDEX_KEYS'] as $noIndexKey)
+				{
+					if (preg_match('/^' . $noIndexKey . '$/', $key))
+					{
+						$noSearch = true;
+						break 2;
+					}
+				}
+			}
+
+			$meta = array
+			(
+				'@context' => 'https://contao.org/',
```

Comment:
Is this correct? Should it not be the current URL?

Reply:
This is correct, imho. It's the software schema and our listener is checking for that to see if it's a relevant schema entry or not:

```php
$jsonLds = $document->extractJsonLdScripts('https://contao.org/', 'PageMetaData');
```

Otherwise it would need to dynamically generate the current URL there as well. That doesn't make sense to me.
If you're adding schema information for phone numbers, addresses etc. you're also referring to to `schema.org` because that's the one defining the schema.